src/components/atoms/venue-square/VenueSquare.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../../theme/Theme.scss';
+
+.venue_square_title {
+    position: relative;
+    margin: auto;
+    clear: left;
+    height: auto;
+    width: 100%;
+
+    font-family: $SUBTITLE-FONT-FAMILY;
+    text-align: center;
+}
+
+.venue-ok {
+    background: $OK;
+}
+
+.venue-closed {
+    background: $FINISHED;
+}
+
+.venue-maintenance {
+    background: $MAINTENANCE;
+}
+
+.venue-unknown {
+    background: $UNKNOWN;
+}
+
+%venue-square {
+    border-width: 1px;
+    box-shadow: lightness($color: #000000);
+    border-style: solid;
+}
+
+.venue_square_collapsed {
+    @extend %venue-square;
+    width: 100%;
+    height: 90%;
+}
+
+.venue_square_collapsed:hover {
+    color: white;
+}
+
+.venue_square_expanded {
+    @extend %venue-square;
+    width: 90%;
+    height: 128px;
+    display: grid;
+    grid-template-rows: 25% 60%;
+    row-gap: 10px;
+}
+
+.icon_box_collapsed {
+    width: 100%;
+    height: 90%;
+    text-align: center;
+    margin-top: auto;
+    background: black;
+}
+
+.icon_box_expanded {
+    height: 60%;
+    display: block;
+    margin-top: 0px;
+    margin-left: auto !important; // Using !important to allow the icon box to be centered.
+    margin-right: auto !important;
+    width: 50%;
+    padding: 50px;
+    text-align: center;
+}
```

Comment:
These should be in IconBox.scss as they are related to the icon boxes, not the venue page.

Reply:
I agree, this wasn't done originally for reasons discussed above.